hosting.md

```diff
@@ -7,19 +7,11 @@ title: Hosting a Review Club Meeting
 
 ## How to Choose a PR
 
-- The purpose of PR Review Club is to help new contributors learn about the
-  Bitcoin Core codebase and the process of contributing to the project. Select a PR
-  that is likely to be instructive or interesting. Don't select a PR just because
-  you want it to be merged soon! You may even select a PR that is already merged.
-
-- The PR can be complex, but keep in mind that attendees may be at a variety of
-  different experience levels. Some attendees have only just cloned the Bitcoin
-  Core repo for the first time, and some have been contributing for months or
-  years. If you choose a complex PR, try to provide notes that would help a
-  beginner get up to speed.
-
-- It's fine to discuss a PR that you opened yourself. After all, you probably
-  know more about it than anyone else!
+- The purpose of PR Review Club is to encourage and enable people to participate
+  in PR review, not just to review things. The best kind of PR is one that is
```

Comment:
"participate in PR review, not just to review things" -> a little ambiguous

(nit, consistent left-margin spacing)